third_party/move/move-compiler/src/parser/keywords.rs

```diff
@@ -52,6 +52,7 @@ pub const CONTEXTUAL_KEYWORDS: &[&str] = &[
     "min",
     "modifies",
     "mut",
+    "package",
```

Comment:
No keyword, please.

Reply:
This is in the list of "contextual" keywords, which includes "schema" for example.